pulsar-broker/src/main/java/org/apache/pulsar/broker/ManagedLedgerClientFactory.java

```diff
@@ -79,21 +79,29 @@ public void initialize(ServiceConfiguration conf, MetadataStoreExtended metadata
         }
 
         statsProvider.start(configuration);
-        StatsLogger statsLogger = statsProvider.getStatsLogger("pulsar_managedLedger_client");
```

Comment:
This is likely to be a breaking change.
I am not against this change but we won't be allowed to cherry pick this to other branches